### PR TITLE
[CBT] Retry flaky tests

### PR DESCRIPTION
**Pull request by ergo14**

### CODE REVIEW

File: timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/LockWatchValueIntegrationTest.java [hunk 1]
```diff
@@ -128,6 +128,7 @@ public void before() {
         LockWatchIntegrationTestUtilities.awaitTableWatched(txnManager, TABLE_REF);
     }
 
+    @ShouldRetry
```
Comment (jeremyk-91) [inline timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/LockWatchValueIntegrationTest.java:131]: I think you can put this at the class-level? (It seems like this is on almost all if not all the methods.) You can override this to be 15 specifically on that one test

**Review by jeremyk-91 — COMMENTED** [review]
I think we can simplify the annotations for LWVIT, unless you've chosen those specific test methods for a reason!

File: timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/LockWatchValueIntegrationTest.java (hunk 1 above)
Comment (ergo14) [reply timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/LockWatchValueIntegrationTest.java]: It is already on this class which I've missed!

Overriding to 15 for all tests given that it retries for all the tests

File: atlasdb-api/src/test/java/com/palantir/atlasdb/keyvalue/api/CellReferenceTest.java [hunk 2]
```diff
@@ -44,6 +50,7 @@ public void dynamicSequentialColumns() {
         assertDistributionUniform(hashes, totalCount);
     }
 
+    @ShouldRetry
     @Test
     public void dynamicRandomColumns() {
         ThreadLocalRandom random = ThreadLocalRandom.current();
```
Comment (ergo14) [inline atlasdb-api/src/test/java/com/palantir/atlasdb/keyvalue/api/CellReferenceTest.java:56]: According to circle, this succeeds 80% of the time

**Review by jeremyk-91 — APPROVED** [review]
👍

**Pull request merged**